index.js

```diff
@@ -61,16 +120,49 @@ const renderHomeMenu = () => {
         innerText: "HOME"
     }, mainElement)
 
-    HOME_MENU_BUTTONS_ARRAY.forEach(({ id, innerText }) => {
+    HOME_MENU_BUTTONS_ARRAY.forEach(({ id, innerText, onclick }) => {
         createElement("button", {
             id,
             className: "home-options",
-            innerText
+            innerText,
+            onclick
         }, mainElement)
     })
 }
 //
 
+// RENDER_INTERNET_CONTENTS
+const renderInternetContents = () => {
+    // createGlobalWrapper()
+    handleOptionsElementOnClick("internetContents")
```

Comment:
It looks like you are reusing code from your other repo. If I was you, I would write new solution instead of coping from somewhere to practice your skills.

Reply:
Tried, wasn't success so I copied from "Iphone"

Reply:
That is not great - in your future you may approach things you cannot copy from somewhere. I suggest you to attempt to write it by yourself and if now working, debug it and resolve bugs

Reply:
I can't find any other way to do it.